### PR TITLE
Update 'contributing.md' to explain how to set up without using ngrok

### PR DESCRIPTION
**Pull request by ldamon**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -19,6 +19,32 @@ Here are a few technical guidelines to follow:
...
+    `./bin/setup`
+
+1. Log into your GitHub account and go to your
+   [application settings].
+
+1. Under the Developer applications panel - Click on "Register new
+   application" and fill in the details:
+
+    * Application Name: Hound Development
+    * Homepage URL: `http://localhost:5000`
```
Comment (jakecraige) [inline CONTRIBUTING.md]: Are you able to authenticate with Github when setting it to this? Sub question, if this works, why are we using ngrok?

Comment (ldamon) [inline CONTRIBUTING.md]: Yes, I authenticated with Github.
@salbertson - can you explain why ngrok is needed?

Comment (gylaz) [inline CONTRIBUTING.md]: @jakecraige ngrok is so GitHub can reach a local machine.

**Comment by gylaz** [comment]
This duplicates a lot of `Configure Hound on Your Local Development Environment` section.

`ngrok` is not a requirement. I was likely a mistake to make it seem that way. We should fix the existing section, and only add the optional `ngrok` steps.

File: CONTRIBUTING.md (hunk 1 above)
Comment (jakecraige) [inline CONTRIBUTING.md]: Ah so you can authenticate via it, but PRs won't trigger a build?

**Comment by ldamon** [comment]
Ok I'll take another stab at this!

File: CONTRIBUTING.md (hunk 1 above)
Comment (salbertson) [inline CONTRIBUTING.md]: Right, authentication works but GitHub cannot make requests via webhook to start a build unless ngrock is used.

**Comment by gylaz** [comment]
:+1:

**Pull request merged**